<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGameWindow</class>
 <widget class="QDialog" name="NewGameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>409</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>718</width>
    <height>409</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>718</width>
    <height>409</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NewGameWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/assets/images/image_3.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
background-image: transparent;
border-image: transparent;</string>
   </property>
   <widget class="QPushButton" name="musicButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 13pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);
</string>
    </property>
    <property name="text">
     <string>Music</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>451</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <widget class="QPushButton" name="mapaUnoButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/assets/images/mapa_1.jpeg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="mapaDosButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/assets/images/mapa_2.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Instant map</string>
    </property>
   </widget>
   <widget class="QComboBox" name="maps">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);
background-color: rgb(68, 68, 68);
background-image: transparent;
border-image: transparent;</string>
    </property>
    <item>
     <property name="text">
      <string>Mapa 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mapa 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mapa 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mapa 4</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>370</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);
background-color: rgb(68, 68, 68);
background-image: transparent;
border-image: transparent;</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>More maps</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="player2Button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Local player 2</string>
    </property>
   </widget>
   <widget class="QComboBox" name="selectPlayers">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);</string>
    </property>
    <item>
     <property name="text">
      <string>1 Player</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 Players</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 Players</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 Players</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
